### ISSUE TITLE
Add to path in petab import

### ISSUE THREAD

**Issue by yannikschaelte** [AMICI-dev/AMICI#1418]
On calls to `amici.petab_import.import_petab_problem` with an existing model, one has to perform
```python
    if output_folder not in sys.path:
        sys.path.insert(0, output_folder)
```
as otherwise an error `Cannot compile to` is thrown. Should the path extension be automatically done in the function for simpler use?

**Comment by dweindl** [comment]
You are saying https://github.com/AMICI-dev/AMICI/blob/9f8fbd870e9267193f2d6d764395a59cee31b5d6/python/amici/petab_import.py#L361-L364 does not work as expected?

**Comment by yannikschaelte** [comment]
I mean the following: On e.g. the Crauste model, run

```python
import amici
import amici.petab_import
import petab

petab_problem = petab.Problem.from_yaml('Crauste_CellSystems2017.yaml')
model = amici.petab_import.import_petab_problem(petab_problem)
print(petab_problem)
```
twice. Then it will compile everything the first time, but the second time the error will throw. I would expect that the function imports the model from existing binary simply then.

**Comment by dweindl** [comment]
Can't seem to reproduce that on current `develop` branch.

**Comment by dweindl** [comment]
P.S.: Is your pre-existing model from an older amici version? That could explain the failure.

**Comment by yannikschaelte** [comment]
I seem to still have had AMICI 11.12 installed. You are right, it is fixed in 11.13 already!

**Comment by yannikschaelte** [comment]
PS Older-version is correctly detected.
